Tests/ProjectSpecTests/SpecLoadingTests.swift

```diff
@@ -41,6 +41,28 @@ class SpecLoadingTests: XCTestCase {
                     Target(name: "NewTarget", type: .application, platform: .iOS, sources: ["template", "target"]),
                 ]
             }
+            
+            $0.context("with optional include for non exist subspecs") {
+                $0.it("ignore subspecs") {
+                    let path = fixturePath + "optional_include/optional_include.yml"
+                    let project = try XCTUnwrap(loadSpec(path: path))
```

Comment:
`XCTUnwrap` will work only on Swift 5.1 or above.

Reply:
Can you use the new `unwrap` function from recently merged https://github.com/yonaskolb/XcodeGen/pull/714

Reply:
fixed